### ISSUE TITLE
[Support Admin] Add organization admin form not showing errors on fields

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#5826]
## Area or Component

[Support Admin > Add organization admin](https://dev.simplereport.gov/app/admin/add-organization-admin)

## Quick Summary

The page does not display errors above the fields when missing. Also, the page doesn't show any error at all if submitted without an organization.

## Accessibility Violation

- [The required field is not identified](https://docs.deque.com/issue-help/1.0.0/en/form-error-required-unclear)
- [Heading inproperly used](https://cdc-devtools.dequecloud.com/issues/91454e38-102a-4b7a-98aa-6a53a7517e70)

## Action Requested

- Use a `<form>` tag 
- Change `<h2>` in the forms to `<legend>`
- Add errors on the required fields when missing

## Screenshots (optional)
![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/05d78fca-ead4-4424-acbe-d33bbbbe795f)
![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/33fdd3ab-c66c-4fbb-9789-9ff5203dc0a9)

**Comment by mehansen** [comment]
look @ using `react-hook-forms` library
